KHJ/14Day/오픈채팅방.js

```diff
@@ -0,0 +1,27 @@
+function solution(record) {
+    var answer = [];
+    const userMap = {}
+    for (let i of record) {
+        let records = i.split(" "); 
+        let say = records[0]
+        let userID = records[1]
+             if (say === "Enter" || say === "Change") {
+                const name = records[2]; 
+                userMap[userID] = name; 
+             }
+    }
+     for (let i of record) {
+        let records = i.split(" "); 
+        let say = records[0]
+        let userID = records[1]
+              
+             if(say === "Enter"){
+                answer.push(`${userMap[userID]}님이 들어왔습니다.`);
+            }else if (say === "Leave"){
+                answer.push(`${userMap[userID]}님이 나갔습니다.`);
+            }
+}
+    return answer;
+}
```

Comment:
```suggestion
   function solution(record) {
    const answer = []; 
    const userMap = {};
    for (let i of record) {
        const records = i.split(" "); 
        const say = records[0];
        const userID = records[1];
             if (say === "Enter" || say === "Change") {
                const name = records[2]; 
                userMap[userID] = name; 
             }
    }
     for (let i of record) {
        const records = i.split(" "); 
        const say = records[0];
        const userID = records[1];

             if (say === "Enter") {
                answer.push(`${userMap[userID]}님이 들어왔습니다.`);
            } else if (say === "Leave") {
                answer.push(`${userMap[userID]}님이 나갔습니다.`);
            }
}
    return answer;
}

```
전반적으로 세미콜론 누락 등등이 많네요 문제는 너무 잘 풀어주셨는데 제출 전 진이 빠지시더라고 한번 더 확인해주시면 좋을 것 같아요 !👍